lib/spack/docs/gpg.rst

```diff
@@ -0,0 +1,43 @@
+.. _gpg:
+
+===========
+GPG support
+===========
+
+Spack has support for signing and verifying packages signed using GPG keys.
+A separate keyring is used for Spack, so any keys available in the user's home
+directory are not used.
+
+When Spack is first installed, its keyring is empty. Keys stored in
+:file:`var/spack/gpg` are the default keys for a Spack installation. These
+keys may be imported by running ``spack gpg init``. This will import the
+default keys into the keyring as trusted keys.
```

Comment:
The previous sentence says `Keys stored in … are the default keys`. Do you have a better wording suggestion that doesn't basically repeat the previous sentence? Or should I just combine them?